NEWS.md

```diff
@@ -3,6 +3,11 @@
 ### Enhancement
 * Provided progress bar for modules loading and data filtering during teal app startup.
 
+### New features
+
+* Footer of `teal` apps is enriched with a button that allows to download the `.lock` file created with `renv` package. 
+This will boost the reproducibility of the app.
```

Comment:
Make news entry as concise as possible and try to avoid stuff that does not bring that much or tries to value changes made.

```suggestion
* Possibility to download `.lock` file to restore app session for reproducibility.
```